pyproject.toml

```diff
@@ -39,6 +40,7 @@ mypy = "^1.0.1"
 black = "^23.1.0"
 pytest-cov = "^4.0.0"
 pdoc = "^13.0.0"
+pytest-dotenv = "^0.5.2"
```

Comment:
if you install this pytest automatically use it or on there is some configuration that tell it to do so?

Reply:
It is automatic no need for config